spark-cassandra-connector/src/it/scala/com/datastax/spark/connector/rdd/CassandraRDDSpec.scala

```diff
@@ -59,10 +59,13 @@ class CassandraRDDSpec extends SparkCassandraITFlatSpecBase {
 
     awaitAll(
       Future {
-        session.execute( s"""CREATE TABLE $ks.short_value (key INT, value SMALLINT, PRIMARY KEY (key))""")
-        session.execute( s"""INSERT INTO $ks.short_value (key, value) VALUES (1,100)""")
-        session.execute( s"""INSERT INTO $ks.short_value (key, value) VALUES (2,200)""")
-        session.execute( s"""INSERT INTO $ks.short_value (key, value) VALUES (3,300)""")
+        if (versionGreaterThanOrEquals(2,2)) {
+          println(s"Found version $cassandraMajorVersion  $cassandraMinorVersion")
```

Comment:
Nah i should drop that println, that's leftover from debugging when I accidentally had my `>` switched with `<`